openedx/adg/lms/applications/api_urls.py

```diff
@@ -0,0 +1,17 @@
+"""
+All urls for applications APIs
+"""
+from django.conf.urls import include
+from django.urls import path
+from rest_framework.routers import DefaultRouter
+
+from .api_views import EducationViewSet, WorkExperienceViewSet
+
+router = DefaultRouter()
+router.register('education', EducationViewSet, basename='education')
+router.register('work_experience', WorkExperienceViewSet, basename='work_experience')
+
+app_name = 'applications'
+urlpatterns = [
+    path('', include(router.urls)),
+]
```

Comment:
We can simply use `urlpatterns = router.urls`. Because we are using include in `lms/urls` or after review changes we will be using include in `adg/lms/urls`.
Here is what Django's documentation says about include
```
Whenever Django encounters include(), it chops off whatever part of the URL matched up to that point and sends the remaining string to the included URLconf for further processing.
```